utils/fo-installdeps

```diff
@@ -12,6 +12,28 @@
 # canonical location for such info, then fix it there first and also
 # update this file and the INSTALL document.
 
+# Line Encoding 
+required_utils=("curl" "dos2unix")
+for util in "${required_utils[@]}"; do
+    if ! command -v $util &> /dev/null; then
+        echo "$util is not installed. Attempting to install..."
+        if [[ $EUID -ne 0 ]]; then
+            echo "Unable to install $util Aborting..."
+            exit 1
+        else
+            case $(uname) in
+                Linux) apt-get install -y $util ;;
```

Comment:
Would recommend checking underlying OS as well to get the correct package manager (as done for other dependencies.)

Reply:
Also, curl will be installed as part of buildtime dependency. Any reason to install it separately here at the beginning of the script?

Reply:
Since, `curl` is installed during the build-time dependencies, there’s no need to include it in this script. I wasn't aware of this before I started modifying the script. I will make this modification in a new commit.